WTWSMS/events/ACR_misc_hellenic_flavor.txt

```diff
@@ -5419,6 +5438,7 @@ character_event = {
 		NOT = { has_character_flag = VIET_artemis_sex_question }
 		NOT = { trait = celibate }
 		NOT = { trait = eunuch }
+		supernatural_events_trigger = yes
 	}
 	
 	weight_multiplier = {
```

Comment:
This event chain does not require lunacy.  In it, Artemis directly saves and heals the player, literally hunts beside him, and they meet fantastical beasts and go on adventures together.

That is supernatural.